test/open_api/decode_yaml_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule OpenApiSpex.OpenApi.DecodYamlTest do
```

Comment:
Can you fix the typo in the module name? Should be `DecodeYamlTest`

Reply:
I don't think issue #359 has anything to do with decoding a YAML file. This test file looks like it's testing the `OpenApiSpex.OpenApi.Decode` module, and there is already a test file (`DecodeTest`) for it.

If you feel that the tests in this file add value to the project, could you move them to the `DecodeTest` file?

The `Decode` module is agnostic to the original file format the api spec was stored in. I don't see a need to bring in the `yamerl` dependency. YAML and JSON parse to identical Elixir data structures.

Reply:
Ok! I've been misleaded by reading the comment on DecodeTest about decomposing and testing also yaml, so I thought to put a skeleton for it, but you're totally right